### PR TITLE
Main

### PR DESCRIPTION
**Pull request by Rushabhshroff**

### CODE REVIEW

File: config/local.json [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+        {
+            "bstack:options": {
+                "os": "Windows",
+                "osVersion": "11"
+            },
+            "browserName": "chrome",
+            "browserVersion": "latest"
+        }
+    ]
+}
```
Comment (Ankit098) [inline config/local.json:24]: EoF fix

File: config/sample.json [hunk 2]
```diff
@@ -20,17 +22,17 @@
...
             "bstack:options": {
-                "osVersion" : "10.0",
-                "deviceName" : "Samsung Galaxy S20"
+                "osVersion": "10.0",
+                "deviceName": "Samsung Galaxy S20"
             }
         }
     ]
-}
+}
```
Comment (Ankit098) [inline config/sample.json:38]: EoF fix

File: README.md [hunk 3]
```diff
@@ -3,26 +3,25 @@
...
 
-## Running your local tests
-* To run a local test, (if you have not set the BROWSERSTACK_ACCESS_KEY environment variable) first go to conftest.py then edit access_key on line 10
+## Run tests on locally hosted websites
+* To run a local test, (if you have not set the BROWSERSTACK_ACCESS_KEY environment variable) first go to config/local.json then edit key on line 3
 * Run `paver run local`
 
-## Running your parallel tests
-* To run parallel tests, run `paver run parallel`
+## Running your sample tests
```
Comment (Ankit098) [inline README.md:20]:
```suggestion
## Run sample tests
```

File: config/sample.json (hunk 2 above)
Comment (Rushabhshroff) [reply config/sample.json]: Done https://github.com/browserstack/pytest-browserstack/pull/5/commits/be53f761257e4e598c88f71638b41ea973b0551f

File: README.md (hunk 3 above)
Comment (Rushabhshroff) [reply README.md]: Done https://github.com/browserstack/pytest-browserstack/pull/5/commits/be53f761257e4e598c88f71638b41ea973b0551f

**Review by Ankit098 — APPROVED** [review]

File: config/local.json [hunk 4]
```diff
@@ -1,22 +1,25 @@
...
+    "capabilities": {
+        "bstack:options": {
+            "projectName": "Pytest Browserstack",
+            "buildName": "browserstack-build-1",
+            "sessionName": "Bstack Local Test",
+            "local": true,
+            "debug": "true",
+            "networkLogs": "true"
+        },
+        "name":"Bstack Local Test"
```
Comment (Ankit098) [inline config/local.json:13]: This can be removed..

**Review by Ankit098 — APPROVED** [review]

**Review by Ankit098 — APPROVED** [review]

**Pull request merged**